server/functions.lua

```diff
@@ -27,77 +15,54 @@ local function getCitizenId(source)
     return player.PlayerData.citizenid
 end
 
----Looking for a vehicle in the world
----@param plate string The plate number of the vehicle.
----@return number? vehicle The entity number of the found vehicle, nil otherwise.
-function public.findVehicleByPlate(plate)
-    local vehicles = GetAllVehicles()
-    for i = 1, #vehicles do
-        local vehicle = vehicles[i]
-        if qbx.getVehiclePlate(vehicle) == plate then
-            return vehicle
-        end
+RegisterNetEvent('QBCore:Server:OnPlayerLoaded', function()
```

Comment:
This is `functions.lua`. Do events & cron jobs belong here?

Reply:
renamed to keys.lua and made it a normal file instead of a module.